src/main/java/racingcar/Car.java

```diff
@@ -1,24 +1,29 @@
 package racingcar;
 
-import java.util.Random;
-
 public class Car {
-    private Random random;
+    private final static int STEP = 1;
```

Comment:
자바의 접근제어자는 아래 순서로 선호하는것이 컨벤션 입니다 😄 
```suggestion
    private static final int STEP = 1;
```
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers